docs-md/concepts/queries/pull.md

```diff
@@ -30,4 +27,40 @@ request/response flows. For asynchronous application flows, see
 Execute a pull query by sending an HTTP request to the ksqlDB REST API, and
 the API responds with a single response.
 
+Pull query features and limitations
+-----------------------------------
+
+- Pull queries are expressed using a strict subset of ANSI SQL.
+- Pull queries are currently available only against materialized aggregate
+  tables, which means tables that are created by using a persistent query
+  with a GROUP BY clause.
+- For non-windowed aggregations, pull queries only support looking up events
+  by key.
+- WHERE clauses can only have `ROWKEY=x`-style bounds for non-windowed tables.
+- Windowed tables support bounds on WINDOWSTART using operators
+  `<=`, `<`, `=`, `>`, `>=`.
+- JOIN, PARTITION BY, GROUP BY and WINDOW clauses aren't supported.
+- SELECT statements can contain column arithmetic and function calls.
+- The result of a pull query isn't persisted anywhere.
+
+Example pull query
+------------------
+
+The following pull query gets all events from `Region_1` that have a timestamp

```

Comment:
Suggestion: change the example to one that gets a users coordinates, so it matches the example picture/use-case above.

```sql
SELECT * FROM user_location
  WHERE ROWKEY = 'user19r7t33'
    AND '2019-10-02T21:31:16' <= WINDOWSTART AND WINDOWSTART <= '2019-10-03T21:31:16';
```

And maybe include some example output?